packages/wonder-blocks-birthday-picker/src/components/__docs__/birthday-picker.argtypes.js

```diff
@@ -2,37 +2,13 @@
 import {defaultLabels} from "../birthday-picker.js";
 
 export default {
-    defaultValue: {
```

Comment:
I noticed that we don't need this anymore, at least for this component.

Reply:
Huh - why is that? Are the descriptions still being generated?

Reply:
Yes :) I'm not sure what happened, but I tried that the first time and didn't work. maybe it's working now that I've fixed the Webpack config (just a random guessing)? I'll check with the Cell docs to see what happens.

https://5e1bf4b385e3fb0020b7073c-bmwroxzzmz.chromatic.com/?path=/docs/birthdaypicker--birthday-picker-default